packages/ui/src/components/va-skeleton/VaSkeleton.demo.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <VaContent>
+    <VbDemo>
+      <VbCard title="Headline" style="min-width: 200px;">
+        <VaSkeleton tag="h1" variant="text" class="va-h1" />
```

Comment:
Can we focus on props in prop demos?

What I'd hope to see is:

* no props (default) select
* demos for variants with minimal amount of additional props/classes
* more rich examples can go to docs, or you can leave one here
* skeleton group could be probably done in different file, given it's a different component
* skeleton group should ideally display component functionality, not idea, so using card component is not needed - it makes it harder to understand what component is doing.